src/styles.scss

```diff
@@ -1,3 +1,6 @@
+$primary: #1C1080;
+$danger: #FF1616;
```

Comment:
We will need to change some of the CSS for the button, like some of the below. Can you please check Figma for the proper styles?
```css
font-weight: bold;
font-size: 14px;
line-height: 21px;
border-radius: 3px;
```

Reply:
Sure thing!

Reply:
Font size is all over the place in Figma. I went with 16px, seemed like the median :)